pages/common/crystal.md

```diff
@@ -0,0 +1,23 @@
+# crystal
+
+> Crystal language tool for compiling.
```

Comment:
We should make this consistent with other pages like `go`. Like - `Tool for managing crystal source code`

Reply:
Fixed with b5de468